src-web/tableDefinitions/utils.js

```diff
@@ -64,7 +66,7 @@ export const transform = (items, def, locale) => {
 }
 
 // use console.log(JSON.stringify(result, circular())) to test return result from transform
-export const transformNew = (items, def, locale) => {
+export const transformNew = (items, def, locale, automationOnClick) => {
```

Comment:
Can you think about another way to do this? This is a little hacky as `automationOnClick` is only used in main table but it's an argument for `transformNew` function. Is it possible to assign this function to table definition for main table?